metron-platform/metron-common/src/main/scripts/stellar

```diff
@@ -38,7 +38,7 @@ set -u
 
 export METRON_PARSERS_PLATFORM="${METRON_PARSERS_PLATFORM:-storm}"
 export HBASE_CONFIGS=$(hbase classpath)
-export STELLAR_LIB=$(find $METRON_HOME/lib/ -name metron-parsers-${METRON_PARSERS_PLATFORM}*.jar)
+export STELLAR_LIB=$(find $METRON_HOME/lib/ -name metron-parsers-*-uber.jar | grep -v "common")
```

Comment:
Could this just be?:
`export STELLAR_LIB=$(find $METRON_HOME/lib/ -name metron-parsers-common-*-uber.jar`

e.g. Running that find command on our parsers dir returns the common uber jar for parsers:
```
find metron-platform/metron-parsing -name metron-parsers-common-\*-uber.jar
metron-platform/metron-parsing/metron-parsers-common/target/metron-parsers-common-0.7.0-uber.jar
```

Reply:
It is my understanding that we don't want common.  We want metron-parsers because that includes metron-parsers-common.

Reply:
The `grep -v` is inverting the search, so that we find anything other than common.

Reply:
And in a follow-on PR, we can probably exclude metron-parsers-common from being deployed.

Reply:
Updated the PR description to better clarify these points.